weaviate/client.py

```diff
@@ -271,6 +271,13 @@ def is_connected(self) -> bool:
         """
         return self._connection.is_connected()
 
+    def is_live(self) -> bool:
+        try:
+            self._connection._ping_grpc()
+        except Exception:
+            return False
+        return super().is_live()
```

Comment:
should we rename this to is_live_http?

Reply:
might break v3 compatibility due to sharing of `_CollectionBase` between v3 and v4 connection objects